scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -109,8 +115,9 @@
                                         "extensionSettings": {
                                             "dataCollectionSettings" : {
                                                 "interval": "[parameters('dataCollectionInterval')]",
-                                                 "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
-                                                 "namespaces": "[parameters('namespacesForDataCollection')]"
+                                                "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
+                                                "namespaces": "[parameters('namespacesForDataCollection')]",
```

Comment:
should these changes also be copied to the ciandsyslogtemplate?

Reply:
we discussed that currently the dataCollectionSettings are not supported for syslog

Reply:
let's discuss it on the sync involving Ganga for the cost optimization part for syslog

Reply:
@wanlonghenry , syslogAndCITemplate is syslog + CI so the data collection settings needs to be copied under CI template snippet of the syslogAndCITemplate  as @pfrcks mentioned.